web/themes/custom/cloudy/templates/block/block.html.twig

```diff
@@ -36,13 +36,7 @@
       'block-' ~ plugin_id|clean_class,
     ]
   %}
-  
```

Comment:
The only issue with removing the bottom margin is that blocks will run up against the element below unless that element has a top margin set. See facets https://powr-1659-portlandor.pantheonsite.io/search/bike

Reply:
Well shoot. I still don't like that we have the bootstrap classes on block level elements like that, but I removed the change for now and we can refactor it in another story.